### ISSUE TITLE
Dashboard/Edit Profile page title does not provide the purpose of the page 

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#13]
**URL**: dashboard: https://hcdev.openplus.ca/en/user/197
edit profile: https://hcdev.openplus.ca/en/user/197/edit
**User agent**: chrome/FF
**Language**: both

**Issue Description:** Page title doesn't describe to the user the purpose/content of the page. Username is not a proper page title for a page. 
**Steps to reproduce:** visual, source code
**Expected results:** All pages should have a page title that describe topic or purpose of the page. Usually the page title and H1 are the same.

**Comment by echouin** [comment]
@mmiron-HC checking w/ OP

**Comment by davidgotlieb** [comment]
Resolved - Titles changed

**Comment by hydegeoff** [comment]
The page titles are now unique, ticket can be closed
